### PR TITLE
mgr/dashboard: Add usage bar component

### PR DESCRIPTION
**Pull request by ricardoasmarques**
This PR adds a component that can be used to display disk usage (e.g. RBD disk usage).

**How to use:**

`<cd-usage-bar [totalBytes]="2048" [usedBytes]="512"></cd-usage-bar>`

screenshot from 2018-03-29 16-29-26

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

**Comment by tspmelo** [comment]
Could you apply this on the Cephfs and Mirroring components? we are using a basic version of the usage bar.

**Comment by ricardoasmarques** [comment]
@tspmelo I'm now using this component on:

**Health page**
screenshot from 2018-04-02 12-05-08

**CephFS**
screenshot from 2018-04-02 12-00-20

**OSDs**
screenshot from 2018-04-02 12-00-37

Mirroring page is not applicable here because this is a "usage bar", not a "progress bar".

**Comment by tspmelo** [comment]
jenkins retest this please

File: src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<ng-template #usageTooltipTpl>
+  <div [innerHtml]="'<table>' +
```
Comment (tspmelo) [inline src/pybind/mgr/dashboard/frontend/src/app/shared/components/usage-bar/usage-bar.component.html]: You can remove this `div` and all the strings. Just use a normal `table` element and use `{{ }}` on the variables.

**Comment by ricardoasmarques** [comment]
@tspmelo I've addressed all your comments.

**Comment by ricardoasmarques** [comment]
jenkins retest this please

**Comment by LenzGr** [comment]
retest this please

**Pull request merged**